pkg/yurtctl/util/kubernetes/util.go

```diff
@@ -77,6 +77,7 @@ const (
 	DisableNodeControllerJobNameBase = "yurtctl-disable-node-controller"
 	// EnableNodeControllerJobNameBase is the prefix of the EnableNodeControllerJob name
 	EnableNodeControllerJobNameBase = "yurtctl-enable-node-controller"
+	DefaultNamespace                = "kube-system"
```

Comment:
My first impression that `DefaultNamespace` is `default`, so how about modify `DefaultNamespace` to `SystemNamespace`?

Reply:
yes, SystemNamespace is more appropriate,I have changed it